### DISCUSSION TITLE
Adding disperion calculation

### DISCUSSION THREAD

**Discussion by jantma2** [mitsuba-renderer/mitsuba3#829] [Feature requests] [answered]
Hello, 

I found this comment in the documentation:

Note

Dispersion is currently unsupported but will be enabled in a future release.

Will this become true?
Thanks for taking care. 

BR

**Comment by njroussel** [comment]
Hi @jantma2 

This comment was already present in Mitsuba 1 over 6 years ago :sweat_smile: .
I'll remove this note from the documentation as there is no plan to add this soon.

**Comment by jantma2** [comment, marked as answer]
No, please don't remove it from the list.
I'm looking for a solution like yours, but with dispersion. Any chance to raise your interest on that topic?
